projects/pillow/fuzz_pillow.py

```diff
@@ -0,0 +1,47 @@
+
+def TestOneInput(data):
+    ImageFile.LOAD_TRUNCATED_IMAGES = True
+    # We need to write a jpeg file first because otherwise libFuzzer will complain that no coverage eixsts
```

Comment:
Wouldn't it be possible to do feed a small image once, and then never do it again?

Reply:
Yup, fixed.